akka-http-core/src/main/resources/reference.conf

```
@@ -21,11 +21,11 @@ akka.http {
     # "PREVIEW" features that are not yet fully production ready.
     # These flags can change or be removed between patch releases.
     preview {
-      # ONLY WORKS WITH `bindAndHandleAsync` (currently)
-      #
       # If this setting is enabled AND the akka-http2-support is found
-      # on the classpath the usual Http().bind... method calls will bind
-      # using HTTP/2. Please note that you must configure HTTPS while doing so.
+      # on the classpath, `Http().newServerAt(...).bind` and `bindSync`
+      # will be enabled to use HTTP/2. Please note that you must configure HTTPS while doing so.
```

```suggestion
      # will be enabled to use HTTP/2.
```

I know this wasn't introduced in this PR, but I don't think this is true anymore.